### DISCUSSION TITLE
NoCredentialProviders: no valid providers in chain v1.5.3

### DISCUSSION THREAD

**Discussion by nikola-vortexwe** [vmware-tanzu/velero#3604] [Community support Q&A] [answered]
**Software information**
OS: CentOS 7
Kubernetes version: v1.19.8
Velero version: v1.5.3
Velero features: None
Velero-for-aws version: v1.1.0 (Tried v1.0.0 and v1.2.0 as well)

**What steps did you take and what happened:**
Velero was installed with the following flags: 
`velero install  --provider aws --namespace velero --bucket velero --plugins velero/velero-plugin-for-aws:v1.1.0 --secret-file ./credentials-velero --use-volume-snapshots=false --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000`
Minio was installed using the example yaml in the repository in the same namespace. The installation process itself runs without issues. However, when deployment's container's logs are checked the following error occurs: 
`level=error msg="Error getting backup store for this location" backupLocation=test controller=backup-sync error="rpc error: code = Unknown desc = NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors" error.file="/go/src/github.com/vmware-tanzu/velero-plugin-for-aws/velero-plugin-for-aws/volume_snapshotter.go:59" error.function=main.getSession logSource="pkg/controller/backup_sync_controller.go:168"`

Upon creating a backup manually, _velero backup describe <backupname>_ returns the information that the backup has failed. _velero backup logs <backupname>_ on the other hand hangs until the request times out. 
Tried reverting to older versions without success. Tried using AWS' S3 which returns the same error. 

**What did you expect to happen:**
Expected for the deployment to run without any issues. 

Let me know if I can provide any further information. Any help is appreciated.

**Comment by ashish-amarnath** [comment, marked as answer]
Based on the error message, the `./credentials-velero` file is not valid aws credentials.
please ensure that the contents of the file are in the below format:
```
[default]
aws_access_key_id = <YOUR_ACCESS_KEY>
aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
```

**Reply by nikola-vortexwe** [reply]
Yeah, figured it out the hard way. Seems that I have omitted to input the _[default]_ entry. It somehow got lost in the redeployment process. Thanks! 
On another note,  [this issue](https://github.com/vmware-tanzu/velero/issues/2942) keeps popping up, although everything is functional.
